lib/ansible/modules/remote_management/ucs/ucs_timezone.py

```diff
@@ -0,0 +1,171 @@
+
+options:
+    state:
+        description:
+            - If C(present), will verify timezone is set (persent) and will set (create) if needed.
+            - If C(absent), will verify timezone is unset (absent) and will unset (delete) if needed.
+        choices: [present, absent]
+        default: absent
+        required: true
+
+    admin_state:
+        description: admin state indicates if the timezone confguration is enabled (utilized by UCS Manager)
+            or disabled (not utilized by UCS Manager)
+        choices: ['disabled', 'enabled']
+        default: "enabled"
+        required: false
+
+    timezone:
+        description:
+            - The timezone name. Time zone names are from the tz database U(https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+            - The timezone name is case sensitive.
+            - This name can be between 0 and 510 alphanumeric characters.
+            - You cannot use spaces or any special characters other than
+            - "\"-\" (hyphen), \"_\" (underscore), \"/\" (backslash)."
+        required: false
+
+    descr:
+        description:
+            - A user-defined description of the timezone object.
+            - Enter up to 256 characters.
+            - "You can use any characters or spaces except the following:"
+            - "` (accent mark), \ (backslash), ^ (carat), \" (double quote), = (equal sign), > (greater than), < (less than), or ' (single quote)."
+        required: no
+
+extends_documentation_fragment:
+    - ucs
+
+requirements:
+
+    - ucsmsdk
+
+author:
+
+    - John McDonough (@movinalot)
+    - CiscoUcs (@CiscoUcs)
+'''
+
+EXAMPLES = r'''
+- name: Configure Time Zone
+  ucs_timezone:
+    hostname: 172.16.143.150
+    username: admin
+    password: password
+    state: present
+    admin_state: enabled
+    timezone: America/Los_Angeles
+    descr: 'Time Zone for Los Angeles'
+
+- name: Unconfigure Time Zone
+  ucs_timezone:
+    hostname: 172.16.143.150
+    username: admin
+    password: password
+    state: absent
+    admin_state: disabled
+'''
+
+RETURN = r'''
+#
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.remote_management.ucs import UCSModule, ucs_argument_spec
+
+
+def main():
+    argument_spec = ucs_argument_spec
+    argument_spec.update(
+        timezone=dict(type='str'),
+        descr=dict(type='str'),
+        admin_state=dict(type='str', default='enabled', choices=['disabled', 'enabled']),
+        state=dict(type='str', default='absent', choices=['present', 'absent']),
+    )
+
+    module = AnsibleModule(
+        argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ['state', 'present', ['timezone']],
+        ],
+    )
+    ucs = UCSModule(module)
+
+    err = False
+
+    # UCSModule creation above verifies ucsmsdk is present and exits on failure, so additional imports are done below.
```

Comment:
I think this comment can be pulled since we don't have any imports for timezone.

Reply:
Pulled